src/js/themes/base.d.ts

```diff
@@ -946,6 +951,25 @@ export interface ThemeType {
     colors?: GraphColorsType;
     extend?: ExtendType;
   };
+  pagination?: {
+    container?: {
+      extend: ExtendType;
+    };
```

Comment:
The container should also be able to accept Box props if I understood correctly.